account_tax_group_widget_base_amount/static/src/xml/tax_group.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates>
+    <t t-extend="AccountTaxGroupTemplate">
+        <t t-jquery="t[t-esc='line[0]']" t-operation="after">
```

Comment:
For clarity, you should only show this if the base is different from the total base. See how is done in the report.

Reply:
Ok, it needed some js override, but now we show it only when there are multiple tax groups

Reply:
Why not using at least `len(lines) > 1` for not overriding JS?

Reply:
Sure. Dumb of me. Done